### ISSUE TITLE
Neo-Geo Battle Coliseum not running on Retropie's Flycast

### ISSUE THREAD

**Issue by luispano** [libretro/flycast#584]
Hi - Has anybody been able to run Neo Geo Battle Coliseum on Retropie's Flycast ? I am getting "Error 01 - This game is not acceptable by main board". After this I press the service button to access the Retroarch settings menu where I am able to test change Regions to USA, Japan or Europe but haven't had any luck.

Also, I made sure to update Retropie's Setup Script before installing lr_flycast. Any recommendations? Thanks in advance

**Comment by flyinghead** [comment]
NeoGeo Battle Coliseum is an Atomiswave game. If you're running a .bin or .dat file then it's likely the problem.

Use a .zip mame rom instead.

**Comment by luispano** [comment]
Hi @flyinghead 

I went ahead and did what you recommended:

1. Copy/pasted my MAME 0.78 rom of ngbc.zip to the Dreamcast folder and set it to run with lr-flycast but unfortunately it doesn't run so I launched it with verbose logging and see a couple of errors which are:

```
-bash: Capabilities:: command not found

-bash: Built:: command not found

-bash: syntax error near unexpected token `('
```

This is the entire log, Do you have any recommendations?

```
Last login: Fri Jul 12 14:40:09 2019 from 192.168.88.225

SSH is enabled and the default password for the 'pi' user has not been changed.
This is a security risk - please login as the 'pi' user and type 'passwd' to set a new password.


     .***.     Friday, 12 July 2019, 14:48:58
     *****     Linux 4.14.30-v7+ armv7l GNU/Linux
     `***'
      |*|      Filesystem      Size  Used Avail Use% Mounted on
      |*|      /dev/root       118G  107G  6.0G  95% /
    ..|*|..    Uptime.............: 0 days, 00h25m24s
  .*** * ***.  Memory.............: 338980kB (Free) / 766776kB (Total)
  *******@@**  Running Processes..: 113
  `*****@@**'  IP Address.........: 192.168.88.109
   `*******'   Temperature........: CPU: 52°C/125°F GPU: 51°C/123°F
     `"""'     The RetroPie Project, https://retropie.org.uk

pi@retropie:~ $   GNU nano 2.7.4         File: /dev/shm/runcommand.log
-bash: GNU: command not found
pi@retropie:~ $
pi@retropie:~ $ Parameters:
-bash: Parameters:: command not found
pi@retropie:~ $ Executing: /opt/retropie/emulators/retroarch/bin/retroarch -L /opt/retro$
-bash: Executing:: command not found
pi@retropie:~ $ [INFO] RetroArch 1.7.6 (Git 9750719)
-bash: syntax error near unexpected token `('
pi@retropie:~ $ [INFO] Redirecting save file to "/home/pi/RetroPie/roms/dreamcast/ngbc.s$
> [INFO] Redirecting savestate to "/home/pi/RetroPie/roms/dreamcast/ngbc.s$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] === Build =======================================
-bash: [INFO]: command not found
pi@retropie:~ $ Capabilities: NEON VFPv3 VFPv4
-bash: Capabilities:: command not found
pi@retropie:~ $ Built: Feb  4 2019
-bash: Built:: command not found
pi@retropie:~ $ [INFO] Version: 1.7.6
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Git: 9750719
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] =================================================
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Loading dynamic libretro core from: "/opt/retropie/libretrocores/$
> [INFO] [overrides] core-specific overrides found at /home/pi/.config/ret$
> [INFO] [overrides] no content-dir-specific overrides found at /home/pi/.$
> [INFO] [overrides] no game-specific overrides found at /home/pi/.config/$
> [INFO] Config: appending config "/home/pi/.config/retroarch/config/Reica$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] [recording] twitch streaming key empty
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: preset directory: /home/pi/.config/retroarch/shaders/pre$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: no game-specific preset found at /home/pi/.config/retroa$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: no game-specific preset found at /home/pi/.config/retroa$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: no content-dir-specific preset found at /home/pi/.config$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: no content-dir-specific preset found at /home/pi/.config$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: no core-specific preset found at /home/pi/.config/retroa$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Shaders: no core-specific preset found at /home/pi/.config/retroa$
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Environ SET_VARIABLES.
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Environ SET_CONTROLLER_INFO.
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO] Controller port: 1
-bash: [INFO]: command not found
pi@retropie:~ $ [INFO]    Controller (ID: 1)
-bash: syntax error near unexpected token `('
pi@retropie:~ $ [INFO]    Arcade Stick (ID: 1025)
-bash: syntax error near unexpected token `('
pi@retropie:~ $ [INFO]    Keyboard (ID: 3)
-bash: syntax error near unexpected token `('
pi@retropie:~ $ [INFO]    Mouse (ID: 2)
-bash: syntax error near unexpected token `('
pi@retropie:~ $ [INFO]    Light Gun (ID: 4)
-bash: syntax error near unexpected token `('
pi@retropie:~ $ [INFO]    Twin Stick (ID: 513)
-bash: syntax error near unexpected token `('
pi@retropie:~ $                            [ Read 233 lines ]
-bash: [: 233: binary operator expected
pi@retropie:~ $ ^G Get Help   ^O Write Out  ^W Where Is   ^K Cut Text   ^J Justify
-bash: :s^G Get Help   ^O Write Out  ^: substitution failed
pi@retropie:~ $ ^X Exit       ^R Read File  ^\ Replace    ^U Uncut Text ^T To Spell
```

I am able to run other Atomiswave roms with .bin + .lst extensions using lr-flycast but Neo-Geo Battle Coliseum doesn't work for some reason.

**Comment by barbudreadmon** [comment]
> Copy/pasted my MAME 0.78 rom of ngbc.zip

1. Use latest `ngbc` rom (aka not a version from 16 years ago)
2. Copy latest `awbios` rom to your flycast system folder (and any other flycast bios you are missing, check https://github.com/libretro/libretro-super/blob/master/dist/info/flycast_libretro.info for full list)

**Comment by luispano** [comment]
Hi @flyinghead , thanks for a well diagnosed problem. Your recommendations worked perfectly. The problem is now resolved.

1. For the BIOS files I ended up putting the ones you listed on the link below into the Dreamcast system folder (Which is the same folder as Flycast).
https://github.com/libretro/libretro-super/blob/master/dist/info/flycast_libretro.info

2. For the roms I used the 0.202 version instead of 0.78.

This webesite and everyting you guys do is awesome, thanks
